wordpress.org/public_html/wp-content/plugins/wporg-bbp-also-viewing/wporg-bbp-also-viewing.js

```diff
@@ -105,17 +121,28 @@
 			return;
 		}
 
-		var offset = 'var(--wp-admin--admin-bar--height, 0px)';
-		if ( 'fixed' === $header.css('position') ) {
-			offset = 'var(--wp-global-header-offset, 0px)';
-		}
-
-		if ( jQuery(window).scrollTop() > bannerOffset ) {
-			banner.css( 'position', 'fixed' );
-			banner.css( 'top', offset );
+		if ( $localNavBar.length ) {
+			// Local nav bar position is set to relative on small screens to stop it being sticky
+			if ( $localNavBar.hasClass( 'is-sticking' ) && $localNavBar.css( 'position' ) !== 'relative' ) {
+				banner.css( 'position', 'sticky' );
+				banner.css( 'top', 'calc(var(--wp-admin--admin-bar--height, 32px) + var(--wp--custom--local-navigation-bar--spacing--height, 60px))' );
```

Comment:
If the admin bar exists, `--wp-admin--admin-bar--height` should be set. If it's not set, there is no admin bar, so the fallback would be `0px`.

Functionally I think the admin bar always exists on Forums, but if something changes, this would cause misalignment.

Reply:
Fixed in [b2bbdc2](https://github.com/WordPress/wordpress.org/pull/201/commits/b2bbdc27e19dde5846d642f09f1d7e1528830c74)